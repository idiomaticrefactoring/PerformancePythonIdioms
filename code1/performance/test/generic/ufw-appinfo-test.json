[{"profile":"TEST","title":"My test app","description":"a longer description of the test app here.","tcp_list":[1,2,3,4,5,6,7,8,9,10,9,8,7,30,53],"tcp_ranges":[{"start":80,"end":90},{"start":8080,"end":8090}],"udp_ranges":[{"start":50,"end":51},{"start":40,"end":60}],"udp_list":[53],"normalized_tcp_list":[1,2,3,4,5,6,7,8,9,10,30,53],"normalized_tcp_ranges":[{"start":80,"end":90},{"start":8080,"end":8090}],"normalized_udp_ranges":[{"start":40,"end":60}]}]
