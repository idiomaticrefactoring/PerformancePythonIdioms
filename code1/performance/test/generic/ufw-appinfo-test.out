Profile: TEST
Title: My test app
Description: a longer description of the test app here.

Ports:
  1,2,3,4,5,6,7,8,9,10,9,8,7,30,80:90,8080:8090/tcp
  50:51,40:60/udp
  53

